services/clusters-keeper/src/simcore_service_clusters_keeper/data/docker-compose.yml

```diff
@@ -72,7 +72,7 @@ services:
     image: ${DOCKER_REGISTRY:-itisfoundation}/autoscaling:${DOCKER_IMAGE_TAG}
     dns: 8.8.8.8 # needed to access internet from AWS
     init: true
-    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    hostname: "{{.Node.Hostname}}"
```

Comment:
Can we keep at least the slot so distinguish among replicas

Reply:
done